llms/mlx_lm/utils.py

```diff
@@ -35,6 +35,8 @@
     lambda m: isinstance(m, nn.Linear)
     and m.weight.shape[0]
     != 8  # avoid quantizing gate layers, otherwise we have to re-quant and upload all the mixtral models
+    and m.weight.shape[0]
+    not in [1, 60] # avoid quantizing qwen2_moe gate layers
```

Comment:
Is it necessary? Would prefer to keep the model consistently quantized to avoid these hacks (the one above is just for back-compatability).

Reply:
I did not quantize those gating layers in the GPTQ-Int4 version either since routing function is sensitive.